src/ThreeEditor/js/Viewport.js

```diff
@@ -144,7 +139,7 @@ export function Viewport(
         if (globalPlane)
             renderer.clippingPlanes = [globalPlane];
 
-        grid.rotation.copy(gridRotation); // update grid rotation
+        grid.rotation.copy(gridRotation ?? new THREE.Euler(0, Math.PI / 2, 0)); // setting default rotation of the grid helper, so it is aligned to the XZ-plane
```

Comment:
Moved here

Reply:
Much better now

Reply:
By the way why do we rotare Y axis by 180 degrees ?

Looking at Euler class documentation second argument is rotation of Y axis.
https://threejs.org/docs/#api/en/math/Euler

I suspect setting this rotation angle to 0 would also yield XZ plane, just from the other side. Is it true ?

Reply:
```suggestion
        grid.rotation.copy(gridRotation ?? new THREE.Euler(0, Math.PI / 2, 0)); // setting default rotation of the grid (90* around Y axis) helper, so it is aligned to the XZ-plane
```